tests/foreman/ui/test_subscription.py

```diff
@@ -337,20 +337,32 @@ def test_select_customizable_columns_uncheck_and_checks_all_checkboxes(session):
     """
     checkbox_dict = {
         'Name': False,
-        'Type': False,
         'SKU': False,
         'Contract': False,
         'Start Date': False,
         'End Date': False,
         'Requires Virt-Who': False,
+        'Type': False,
```

Comment:
Does it make a difference? If not, why did you decide to change it?

Reply:
no impact here, I will change too as it is